README.md

```diff
@@ -1,33 +1,26 @@
-# councildataproject.github.io / councildataproject.org
+# councildataproject.org
 
-[![Deployment Status](https://github.com/CouncilDataProject/councildataproject.github.io/workflows/Build%20and%20Deploy/badge.svg)](https://github.com/CouncilDataProject/councildataproject.github.io/actions)
+[![Deployment Status](https://github.com/CouncilDataProject/cdp-homepage/workflows/Build%20and%20Deploy/badge.svg)](https://github.com/CouncilDataProject/cdp-homepage/actions)
```

Comment:
repo name is still `councildataproject.github.io` not `cdp-homepage`.

Reply:
Whoops thanks for the catch!